build.gradle

```diff
@@ -40,7 +40,7 @@ targetCompatibility = 1.7
 ext {
   generatedSrcDir = "${projectDir}/generated/src"
 
-  grpcVersion = '1.0.3'
+  grpcVersion = '1.2.0'
```

Comment:
@pongad I plan to regenerate and bump up the grpc version on April 5 or 6. (Right now I am stuck with a header fix)